File: backend/app/schemas/stats.py
"""Schémas pour les statistiques (dashboard overview)."""

from pydantic import BaseModel


class StatsOverview(BaseModel):
    """KPIs du dashboard."""

    revenue_today: int
    orders_new: int
    orders_preparing: int
    orders_ready: int
    reservations_upcoming: int
    events_pending_deposit: int

File: src/api/stats.ts
// API stats : KPIs dashboard.

import { apiGet } from './client'

export interface StatsOverview {
  revenue_today: number
  orders_new: number
  orders_preparing: number
  orders_ready: number
  reservations_upcoming: number
  events_pending_deposit: number
}

export async function getStatsOverview(): Promise<StatsOverview> {
  return apiGet<StatsOverview>('/stats/overview')
}
